content/blog_posts/articles/whats-coming-next-adonisjs-6.md

```diff
@@ -0,0 +1,232 @@
+---
+summary: Our latest experimentation and future iterations for AdonisJS 6
+---
+
+We have just released AdonisJS 6 with a ton of new features and improvements. This release was a big milestone for us as we have been working on it for the last year. But we are not done yet and still got lots of ideas in the pipeline.
+
+In order to finally release AdonisJS 6, we had to put on hold some ideas that we have been experimenting with. Now that AdonisJS 6 is out, our immediate focus is going to be on stabilizing it, fixing bugs and improving the documentation and migration guide. But we'll soon be able to get back to our ideas.
+
+Last year, we've spent a lot of time testing new ideas and features. Some of them are already in pretty good state, while others are still being thought and worked on. In this article, I'm going to share some of those ideas.
+
+**Please, do not expect or ask for any release dates or timelines. We will share more details when we are ready.**
+
+## Migrating last few packages to AdonisJS 6
+
+First focus will be to migrate the last few packages to AdonisJS 6. We can't commit on a specific date, but expect them to be migrated in the next few weeks :
+
+- Drive
+- Limiter
```

Comment:
Let's add the link to release notes here. Saying "Limiter (Already released)" https://github.com/adonisjs/limiter/releases/tag/v2.1.0